kubernetes/zenko/charts/backbeat/templates/replication/data_processor_deployment.yaml

```diff
@@ -98,6 +98,8 @@ spec:
               value: "{{ template "backbeat.redis-hosts" . }}"
             - name: REDIS_HA_NAME
               value: "{{ .Values.redis.sentinel.name }}"
+            - name: EXTENSIONS_REPLICATION_QUEUE_PROCESSOR_RETRY_TIMEOUT_S
```

Comment:
Depending on the user's needs, it seems like we might as well introduce `EXTENSIONS_REPLICATION_REPLICATION_STATUS_PROCESSOR_RETRY_TIMEOUT_S` as another tunable because we have that option exposed now.